Sources/Logging/Documentation.docc/Logging.md

```diff
@@ -0,0 +1,30 @@
+# ``Logging``
```

Comment:
how about we standardize on `Sources/<Module>/Documentation.docc/index.md` or `Sources/<Module>/Docs.docc/index.md`  for the "landing page"

Reply:
I'll check if that works... docc has a lot of weird assumptions about "same name".

Filename of the md file can be different indeed though -- index.md could be good! 👍

Reply:
Ok yeap that works -- sounds good to me.